ntfyme/utils/telegram/telegram.py

```diff
@@ -0,0 +1,22 @@
+
+import requests
+import toml
+import os
+USER_NAME = "https://t.me/notify_cli_bot"
+TOKEN = "7306967185:AAHnM_GHZwAAF8XGInfejGk00VqWtEfL4kQ"
+
+def send_gmail(result):
+    current_dir = os.path.dirname(__file__)
+    package_root = os.path.abspath(os.path.join(current_dir, '..'))
+    config_path = os.path.join(package_root, 'config.toml')
+
+    with open(config_path, 'r') as file:
+
+        data = toml.load(file)
```

Comment:
do not use the word "data", please use words like "details" or "info" or similar